### PR TITLE
Add Fix functions from Catmandu Cheat Sheet.

### LINKED ISSUE #100: Add more Fix functions.
We're going to need some more Fix functions that aren't implemented yet. E.g.:

| Morph     | Fix           |
|-----------|---------------|
| `replace` | `replace_all` |
| `split`   | `split_field` |
| `unique`  | `uniq`        |

I'm just going to go ahead and implement those plus a bunch more from the [Catmandu Cheat Sheet](https://librecat.org/assets/catmandu_cheat_sheet.pdf).

Functional review: @TobiasNx
Code review: @fsteeg

### PR DESCRIPTION
**Pull request by blackwinter**
I've opted for the non-destructive approach w.r.t. `Value`s. I don't think we're consistent as to when we modify an existing value and when we create a new one. Let me know if you have a preference - or if I'm mistaken and there's indeed a system to it ;)

Omitted Fix functions from the [cheat sheet](https://librecat.org/assets/catmandu_cheat_sheet.pdf):

* `assoc()`
* `diassoc()`
* `compact()` (not applicable)
* `flatten()`
* `from_json()`
* `to_json()`
* `uri_decode()`
* `uri_encode()`

Resolves #100.

### CODE REVIEW

**Review by fsteeg — APPROVED** [review]
:raised_hands:

**Comment by blackwinter** [comment]
@fsteeg: Do you want to re-review now that there have been substantial changes based on the discussion in #100?

File: metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
     private static final String CSV_MAP = "src/test/resources/org/metafacture/metafix/maps/test.csv";
     private static final String TSV_MAP = "src/test/resources/org/metafacture/metafix/maps/test.tsv";
 
+    private static final String LOOKUP = "lookup('title.*',";
```
Comment (fsteeg) [inline metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java:42]: Minor: for tests, I often like to keep redundancies if they add clarity, to get kind of contained test cases.

**Review by fsteeg — APPROVED** [review]

File: metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java (hunk 1 above)
Comment (blackwinter) [reply metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java:42]: Well, what can I say? Got overpowered by my OCD ;)

**Pull request merged**